index.html

```diff
@@ -13,19 +13,30 @@
 
         <div class="container">   
 
-            <!-- the header section   -->
+            <nav class="mobile-modal close">
+                <img id="menu-exit" class="close-btn" src="images/Icon.png" alt="">
+              
+                <ul id="mobile-links" class="mobile menu-bar">
+                    <li> <a href="#contact">Contact</a></li>
+                    <li> <a href="#about">About</a></li>
+                    <li> <a href="#portfolio">Portfolio</a></li>
+                </ul>
```

Comment:
- [ ] Kindly try to follow the design on Figma as much as possible. This includes the order of the links, the `opacity` of the menu (it's a bit transparent on Figma). Also, kindly remove the white spaces around the menu 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/196508794-4e3f325c-f4ff-43bd-a3d9-721fb005d23c.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/196508838-52225833-97fd-48bf-868e-555dcac2d962.png)

Reply:
Hi @ha-manel 

Thank you so much for the review.

I have made the required changes.

I will be grateful if you share your opinion about it.

Thank you